src/h5web/vis-packs/core/shared/TooltipMesh.module.css

```diff
@@ -1,6 +1,12 @@
 .tooltip {
   color: inherit !important;
-  background-color: rgba(245, 251, 239, 0.7) !important;
+  background-color: var(--primary-light-bg) 0.7 !important;
```

Comment:
Actually, this is not exactly the same color but quite close: https://calcolor.co/blend/fafdf7/f5fbef